Sources/PointFree/Account/Cancel.swift

```diff
@@ -73,23 +75,23 @@ private func cancel(_ conn: Conn<StatusLineOpen, Tuple2<Stripe.Subscription, Dat
               to: .account(.subscription(.changeSeats(.show))),
               headersMiddleware: flash(.error, "We couldn’t cancel your subscription at this time.")
             )
-          ),
-          const(
-            conn |> redirect(
-              to: .account(.index),
-              headersMiddleware: flash(.notice, "We’ve canceled your subscription.")
-            )
           )
-        )
+        ) { _ in
```

Comment:
whoa my brain is having a hard time parsing this. is this a trailing closure for a curried `either` or something?

Reply:
Ah! It's `either(L -> A, B -> A)`. The left path is a const so I pass it through. I'm using a trailing closure for the "right" path.